src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Comments } from '../../types/Comments/Comments';
+
+import './CommentInfo.scss';
+
+interface Props {
+  commentItem: Comments;
```

Comment:
```suggestion
  commentItem: Comment;
```
It should be singular, not plural